cumulusci/tasks/metadata_etl/help_text.py

```diff
@@ -0,0 +1,87 @@
+from collections import defaultdict
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.tasks.metadata_etl import MetadataSingleEntityTransformTask
+from cumulusci.utils.xml.metadata_tree import MetadataElement
+from cumulusci.core.utils import process_list_arg, process_bool_arg
+
+
+class SetFieldHelpText(MetadataSingleEntityTransformTask):
+    entity = "CustomObject"
+    task_options = {
+        "fields": {
+            "description": "List of object fields to affect, in Object__c.Field__c form.",
+            "required": True,
+        },
+        "overwrite": {
+            "description": "List of object fields to affect, in Object__c.Field__c form."
```

Comment:
@jofsky can you update this option's description please?

Also run `make docs` to regenerate docs/tasks.rst, and make sure that the section for this task is accurate and complete.

Reply:
Docs revised in 52bd579c